Detectors/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ add_subdirectory(PHOS)
 add_subdirectory(MUON)
 add_subdirectory(ZDC)
 add_subdirectory(GlobalTracking)
+add_subdirectory(Calibration)
```

Comment:
@shahor02 suggested to put it on a higher level, since it concerns not only TPC, but also ITS, TRD and TOF. Only that part using the other detectors is still missing and needs to be added. A description of the task is also here: https://alice.its.cern.ch/jira/browse/O2-361

Reply:
Indeed, since this class will depend also on other detector libs, I did not want to make TPC to depend on it. ``Detectors/TPC/calibration`` compiled to separate library would also do this, we can move it there.

Reply:
Hi @shahor02 
Should I move everything then to `Detectors/TPC/calibration`? I am not sure what I have to put in `CMakeLists.txt` there such that it compiles to a separate library..
But that would make it easier for me to use the already defined TPC parameters from the headers @wiechula mentioned, since they are in namespace `o2::TPC`.

Reply:
Hi @martenole, I think you simply need to ``cd O2; git mv Detectors/Calibration Detectors/TPC/calibration`` 
and move the line ``add_subdirectory(Calibration)`` from the ``Detectors/CMakeLists.txt`` to ``Detectors/TPC/CMakeLists.txt``. Creation of separate spacepoints calibration library will be steered by the ``O2_GENERATE_LIBRARY()`` in the ``CMakeLists.txt`` of the ``SpacePoints``. Later we can decide if we keep a separate library for this calibration only or put all TPC calibs. to the same lib.

Reply:
Ok, thanks it was quite easy with that command. I have now moved everything to `Detectors/TPC/calibration`.